internal/cli/project_maintenance_test.go

```
@@ -0,0 +1,112 @@
+	if err := (config.AtomicWriter{}).Write(configPath, config.Config{
+		Version:  config.Version,
+		Projects: []config.Source{{Path: canonical}},
+	}); err != nil {
+		t.Fatal(err)
+	}
+	return canonical, configPath
+}
+
+func runTestGit(t *testing.T, dir string, args ...string) {
+	t.Helper()
+	command := exec.Command("git", args...)
```

golangci-lint (2.13.2)

[error] os/exec.Command must not be called. use os/exec.CommandContext

(noctx)